app/src/main/java/org/mozilla/fenix/components/Analytics.kt

```diff
@@ -117,12 +118,15 @@ class Analytics(
                 // from the app unless the user does so from a UI control.
                 // However, the user may have opt-ed out of mako experiments already, so
                 // we should respect that setting here.
-                val enabled = context.settings().isExperimentationEnabled
+                val enabled = context.components.strictMode.resetAfter(StrictMode.allowThreadDiskReads()) {
+                    context.settings().isExperimentationEnabled
+                }
                 if (!enabled) {
                     globalUserParticipation = enabled
                 }
-                context.settings().isExperimentationEnabled = globalUserParticipation
```

Comment:
Once this is set first time, this will be kept in sync, so we don't need this read here.